src/bootstrap/src/core/build_steps/run.rs

```diff
@@ -135,29 +133,35 @@ impl Step for Miri {
     }
 
     fn make_run(run: RunConfig<'_>) {
-        run.builder.ensure(Miri {
-            stage: run.builder.top_stage,
-            host: run.build_triple(),
```

Comment:
`run.build_triple()` returns `builder.cmd.build`; I assume that's always the same as `builder.build.build`?

Reply:
> run.build_triple() returns builder.cmd.build

`run.build_triple()` returns `builder.build.build` already.

Reply:
Ah yes, you are right... and `builder.cmd.build` does not exist.
But some places use `builder.config.build`. That is hopefully the same as `builder.build.build`.

Reply:
Yes, they are identical.